applicationconfig/client/pom.xml

```diff
@@ -3,88 +3,88 @@
-    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-app-configuration</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
 
-    <distributionManagement>
-        <site>
-            <id>azure-java-build-docs</id>
-            <url>${site.url}/site/${project.artifactId}</url>
-        </site>
-    </distributionManagement>
+  <name>Microsoft Azure client library for Application Configuration</name>
+  <description>This package contains the Microsoft Azure Application Configuration client library.</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
 
-    <scm>
-        <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
-        <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
-        <tag>HEAD</tag>
-    </scm>
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>properties-maven-plugin</artifactId>
-                <version>1.0.0</version>
-                <executions>
-                    <execution>
-                        <phase>generate-resources</phase>
-                        <goals>
-                            <goal>write-project-properties</goal>
-                        </goals>
-                        <configuration>
-                            <outputFile>
-                                ${project.build.outputDirectory}/maven.properties
-                            </outputFile>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <tag>HEAD</tag>
+  </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>com.azure</groupId>
-            <artifactId>azure-common</artifactId>
-        </dependency>
+  <build>
+    <plugins>
+      <!-- Generates resources that are read during testing. -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>properties-maven-plugin</artifactId>
+        <version>1.0.0</version>
+        <executions>
+          <execution>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>write-project-properties</goal>
+            </goals>
+            <configuration>
+              <outputFile>
+                ${project.build.outputDirectory}/maven.properties
+              </outputFile>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Sorry if I'm dense, but didn't I ask if this is necessary? Did you clarify if it was (or if it could be promoted into pom.client.xml)? I like seeing as little config as possible in any pom.